types/yargs-parser/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for yargs-parser 11.0
+// Project: https://github.com/yargs/yargs-parser#readme
+// Definitions by: Miles Johnson <https://github.com/milesj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare namespace yargsParser {
+    // Copied from @types/yargs
```

Comment:
Rather than copy these, you can say

```ts
import {Arguments as YargsArgs} from "yargs";

declare namespace yargsParser {
  export import Arguments = YargsArgs;
}
```
(and similar for other types) to avoid the copying.

Reply:
@weswigham Done